include/boost/gil/image_processing/scaling.hpp

```diff
@@ -5,6 +5,29 @@
 
 namespace boost{
 namespace gil{
+/// \defgroup ScalingAlgorithms
+/// \brief Algorthims suitable for rescaling
+///
+/// These algorithms are used to improve image
```

Comment:
@simmplecoder You may go for longer lines, up to 90 characters :-)

Reply:
@mloskot , thanks, helped on one occasion, but unfortunately there was a place where it still was too long. Resolving this.